### PR TITLE
Add required /fetch headers.

### PR DESCRIPTION
**Pull request by RobBoeckermann**
Fix /fetch json syntax.

Here's the current documentation: https://docs.circonus.com/irondb/api/data-retrieval/#retrieving-and-transforming-data

Header information has not been specified in this documentation before, so there was not a reference for how to format them. How does this look?
image

### CODE REVIEW

File: content/irondb/api/data-retrieval.md [hunk 1]
```diff
@@ -110,6 +110,15 @@ It allows for fetch submissions in both FlatBuffers and JSON formats, and return
 
 POST
 
+#### Headers
+```
+"accept: <payload_type>"
+
+"Content-Type: <payload_type>"
```
Comment (bdunavant) [inline content/irondb/api/data-retrieval.md:117]:
```suggestion
"accept: <desired_response_type>"

"Content-Type: application/json"
```

Comment (RobBoeckermann) [reply content/irondb/api/data-retrieval.md:117]: @bdunavant shoud I remove `<payload_type> ::= "application/json" | "x-irondb-fetch-flatbuffer"` then?

Comment (bdunavant) [reply content/irondb/api/data-retrieval.md:117]: Yeah, probably better to just leave this in swagger.

Comment (RobBoeckermann) [reply content/irondb/api/data-retrieval.md]: okay. this ticket/PR will just fix the json formatting 👍

**Pull request merged**